Assets/Scripts/GUI/AppSettingsPanel.cs

```diff
@@ -28,9 +28,37 @@ public override void InitPanel()
             m_ToggleExperimentalModeToggle.m_IsToggledOn = App.Config.GetIsExperimental();
         }
 
+        public void HandleToggleHandedness(ToggleButton btn)
+        {
+            InputManager.m_Instance.WandOnRight = !btn.m_IsToggledOn;
+        }
+
+        public void HandleResetFirstUse()
+        {
+            if (!PanelManager.m_Instance.AdvancedModeActive())
```

Comment:
I didn't test this, but is this condition correct? Should the `!` be there?

Reply:
Fixed this on 192301a